### DISCUSSION TITLE
List of all supported Named Entities

### DISCUSSION THREAD

**Discussion by eladbitton** [explosion/spaCy#9147] [Help: Coding & Implementations] [answered]
I am using spacy for NER in multiple languages.

Where can I find a list of all supported named entity labels supported in spacy ner models?
Can't find it in the docs.

**Comment by polm** [comment, marked as answer]
The NER label scheme varies by language, and depends heavily on what kind of training data was available. You need to check the "Label Scheme" entry on the model page, which should have an NER section. For example, [here's Japanese][ja]. 

[ja]: https://spacy.io/models/ja#ja_core_news_sm-labels

**Reply by eladbitton** [reply]
Thanks for the reply. I gathered all the labels that I need.

This is the full list of labels I need:
```
"CARDINAL", 
        "DATE", 
        "EVENT", 
        "FAC", 
        "GPE", 
        "LANGUAGE", 
        "LAW", 
        "LOC", 
        "MONEY", 
        "NORP", 
        "ORDINAL", 
        "ORG", 
        "PERCENT", 
        "PERSON", 
        "PRODUCT", 
        "QUANTITY", 
        "TIME", 
        "WORK_OF_ART",
        "LOC", 
        "MISC", 
        "ORG", 
        "PER"
```

Some explanations I found for these:
```
PERSON:      People, including fictional.
NORP:        Nationalities or religious or political groups.
FAC:         Buildings, airports, highways, bridges, etc.
ORG:         Companies, agencies, institutions, etc.
GPE:         Countries, cities, states.
LOC:         Non-GPE locations, mountain ranges, bodies of water.
PRODUCT:     Objects, vehicles, foods, etc. (Not services.)
EVENT:       Named hurricanes, battles, wars, sports events, etc.
WORK_OF_ART: Titles of books, songs, etc.
LAW:         Named documents made into laws.
LANGUAGE:    Any named language.
DATE:        Absolute or relative dates or periods.
TIME:        Times smaller than a day.
PERCENT:     Percentage, including ”%“.
MONEY:       Monetary values, including unit.
QUANTITY:    Measurements, as of weight or distance.
ORDINAL:     “first”, “second”, etc.
CARDINAL:    Numerals that do not fall under another type.
```
[Credit](https://towardsdatascience.com/explorations-in-named-entity-recognition-and-was-eleanor-roosevelt-right-671271117218)

I believe if these labels were found somewhere convenient in the docs that would have helped users a lot when using spacy for ner.

**Reply by adrianeboyd** [reply]
Hi, spacy includes definitions for common labels under `spacy.explain(label)`, see: https://spacy.io/usage/linguistic-features#entity-types
